Formula/mame.rb

```diff
@@ -38,16 +37,18 @@ def install
     # Cut sdl2-config's invalid option.
     inreplace "scripts/src/osd/sdl.lua", "--static", ""
 
-    system "make", "USE_LIBSDL=1",
+    # Use bundled asio and lua instead of latest version.
```

Comment:
If it can use the brewed version it should use that.

Reply:
No, it can't.
See #48102 and #57133.